examples/splinterdb_admin_config_example.c

```diff
@@ -0,0 +1,135 @@
+// Copyright 2022 VMware, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+/*
+ * SplinterDB Basic Configuration Example Program.
+ */
+
+#include <stdio.h>
+#include <string.h>
+
+#include "splinterdb/default_data_config.h"
+#include "splinterdb/splinterdb.h"
+#include "example_common.h"
+
+/* Tag to identify messages from application program */
+#define APP_ME "App-Config"
+
+/*
+ * App-specific 'defaults' that can be parameterized, eventually.
+ */
+#define APP_DB_NAME "splinterdb_admin_example_db"
+
+#define APP_DEVICE_SIZE_MB 1024 // Size of SplinterDB device; Fixed when created
+#define APP_CACHE_SIZE_MB  64   // Size of cache; can be changed across boots
+
+/* Application declares the limit of key-sizes it intends to use */
+#define APP_MAX_KEY_SIZE ((int)100)
+
+// Function Prototypes
+void
+configure_splinter_instance(splinterdb_config *splinterdb_cfg,
+                            data_config       *splinter_data_cfg,
+                            const char        *filename,
+                            uint64             dev_size,
+                            uint64             cache_size);
+
+int
+reboot_splinter_instance(const splinterdb_config *splinterdb_cfg,
+                         splinterdb             **spl_handle_out);
+
+/*
+ * -------------------------------------------------------------------------------
+ * main() Driver for basic SplinterDB example program.
+ * -------------------------------------------------------------------------------
+ */
+int
+main()
+{
+   printf("     **** SplinterDB Basic Example program ****\n\n");
+
+   // Initialize data configuration, describing your key-value properties
+   data_config splinter_data_cfg;
+   default_data_config_init(APP_MAX_KEY_SIZE, &splinter_data_cfg);
+
+   // Basic configuration of a SplinterDB instance
+   splinterdb_config splinterdb_cfg;
+   configure_splinter_instance(&splinterdb_cfg,
+                               &splinter_data_cfg,
+                               APP_DB_NAME,
+                               (APP_DEVICE_SIZE_MB * K_MiB),
+                               (APP_CACHE_SIZE_MB * K_MiB));
+
+   splinterdb *spl_handle = NULL; // To a running SplinterDB instance
+
+   // -- ACTION IS HERE --
```

Comment:
Each program "builds on top" of the previous one, in some ways.

I am using this `-- ACTION IS HERE --` comment tag to direct the reader to parts of the program that is new and different than its previous on the list of programs in `README.md`.

(I don't know of any other way to high-light the new-ness in each program ... Maybe **BOLD ***** ...** but I thought that may be too too distracting.)